src/Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileImage/Components/LocalImagePreview.tsx

```diff
@@ -0,0 +1,18 @@
+import { Avatar } from "@artsy/palette"
+import { Media } from "Utils/Responsive"
+
+export const LocalImagePreview: React.FC<{ imageUrl: string }> = ({
+  imageUrl: imageUrl,
+}) => {
+  return (
+    <>
+      <Media lessThan="sm">
+        <Avatar src={imageUrl} size="xs" lazyLoad mr={[1, 2]} />
```

Comment:
**suggestion:** have you considered the following to avoid media queries
```suggestion
        <Avatar src={imageUrl} size=["xs", "md"] lazyLoad mr={[1, 2]} />
```